databrowser/src/domain/cellComponents/plugins/registerCellComponents.ts

```diff
@@ -45,5 +50,7 @@ export default {
     app.component(CellComponent.WebcamGalleryCell, WebcamGalleryCell);
 
     app.component(FilterComponent.FixedValue, FixedValue);
+
+    app.component(CellComponent.ArrayCellTags, ArrayCellTags);
```

Comment:
The components are ordered by:
1. type (Cell, Filter)
2. name

The components are ordered by name, please apply